src/common/mod.rs

```diff
@@ -0,0 +1,38 @@
+//! Common helpers for `displayz`
+
+/// Height and Width of a Display (`usize`)
+pub type Resolution = (usize, usize);
+
+/// X/Y positions of a display.
+pub type Position = (usize, usize);
```

Comment:
I would use i32 here, sinze position can also be negative

Reply:
I've changed both to a `i32`, and they're now structs. Will do an `impl` later on.

Reply:
Fixed struct/`i32` in 15bb73f.